src/BugsnagUnity/Native/Cocoa/NativeClient.cs

```diff
@@ -34,6 +34,10 @@ IntPtr CreateNativeConfig(IConfiguration config)
             NativeCode.bugsnag_setAppVersion(obj, config.AppVersion);
             NativeCode.bugsnag_setNotifyUrl(obj, config.Endpoint.ToString());
             NativeCode.bugsnag_setMaxBreadcrumbs(obj, config.MaximumBreadcrumbs);
+            if (!string.IsNullOrEmpty(config.BundleVersion))
```

Comment:
This if statement doesn't seem to be necessary - we don't check `IsNullOrEmpty` for `AppVersion` or `ReleaseStage`

Reply:
The (internal) spec says:
> The `bundleVersion` property shall be set from supplied `Configuration`.

So yes, I agree - if it's null/empty then it should be set as such in the native layer.